### ISSUE TITLE
default aria-valuenow for scrollbar

### ISSUE THREAD

**Issue by kbae00** [w3c/aria#704]
Currently the [1.1 scrollbar spec](https://www.w3.org/TR/wai-aria/#scrollbar) says "If aria-valuenow is missing or not a number, it defaults to the value half way between aria-valuemin and aria-valuemax." But it seems like it should default to 0 instead as it's most likely the scrollbar will be at the top when scrollable content is first encountered.

**Comment by jnurthen** [comment]
aria-valuenow is now required on scrollbar. Closing.
